docs/trying_pinot.rst

```diff
@@ -1,5 +1,5 @@
-Running the Pinot Demonstration
-===============================
+Quickstart guide
```

Comment:
Quick start somewhat implies that this will get you to the final, production-ready state of the system. Think of the "quick start" instructions you get when buying something, they usually consist of getting the system up in the fastest way.

Reply:
Are you suggesting we go back to the 'demo' title?